commander/src/bootstrapping/templates/lisk-template-ts/templates/asset/src/app/modules/assets/asset.ts

```diff
@@ -13,11 +13,12 @@ export class <%= assetClass %> extends BaseAsset {
 		properties: {},
   };
 
-    public validate({ asset }: ValidateAssetContext<{}>): void {
+  public validate({ asset }: ValidateAssetContext<{}>): void {
     // Validate your asset
   }
 
-  	public async apply({ asset, transaction, stateStore }: ApplyAssetContext<{}>): Promise<void> {
+	// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
i don't think we need this?

Reply:
In bootstrapped application its cause linting errors.